pages/search/search.jsx

```diff
@@ -34,7 +29,7 @@ export default React.createClass({
       // make sure input box value is updated with searchQueryInUrl
       // then use searchQueryInUrl as the search param to fetch data from Pulse API
       this.refs.searchInput.setState({value: searchQueryInUrl ? decodeURIComponent(searchQueryInUrl) : ``});
```

Comment:
This feels wrong. State is private, we should not be messing with another component's state. If we want their values to change, in the `render()` function make sure to pass the new value as a prop, and make that component do the right thing when it sees a property update.